cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -5,7 +5,20 @@ service:
       receivers: [otlp, jaeger, zipkin]
       processors: [batch]
       exporters: [jaeger_storage_exporter]
-
+  telemetry:
```

Comment:
this is just one of many config files, we need to make similar changes to all of them, and to validate that the unique metrics produced by each binary/config are the same as in v1.

Reply:
Sure, that can be taken as the next step.